server/src/main/java/com/ahmadda/domain/Guest.java

```diff
@@ -38,12 +38,17 @@ private Guest(final Event event, final OrganizationMember participant) {
 
         this.event = event;
         this.participant = participant;
+        event.getGuests().add(this);
     }
 
     public static Guest create(final Event event, final OrganizationMember participant) {
         return new Guest(event, participant);
     }
 
+    public boolean isSameGuest(final OrganizationMember organizationMember) {
+        return this.participant.equals(organizationMember);
+    }
```

Comment:
p3.
isSameGuest보다는 isSameParticipant가 더 자연스럽지 않나요?
```suggestion
    public boolean isSameParticipant(final OrganizationMember participant) {
        return this.participant.equals(participant);
    }
```